pkg/plugins/mellanox/mellanox_plugin.go

```diff
@@ -171,6 +171,11 @@ func (p *MellanoxPlugin) OnNodeStateChange(new *sriovnetworkv1.SriovNetworkNodeS
 	return
 }
 
+// OnNodeStatusChange verify whether SriovNetworkNodeState CR status present changes on configured VFs.
+func (p *MellanoxPlugin) CheckStatusChanges(*sriovnetworkv1.SriovNetworkNodeState) (bool, error) {
+	return false, nil
```

Comment:
Hi please a TODO and comment here that we skip the externallyManage with a link to the issue we open on the repo please